docs/guides/qiskit-addons-cutting-wires.ipynb

```diff
@@ -0,0 +1,552 @@
+{
```

Comment:
<p>I'd prefer not to refer to it as automatic wire cutting.  I don't want it to be confused with the automated cut finder, which is some entirely different feature of the package.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<blockquote>The code blocks below performs</blockquote><p>one of these words needs its trailing s removed</p>